main.py

```diff
@@ -1,10 +1,44 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        players_data = json.load(f)
+
+    for key in players_data:
+        race, created = Race.objects.get_or_create(
+            name=players_data[key]["race"]["name"],
+            defaults={"description": players_data[key]["race"].get(
+                "description", "")})
+        for elem in players_data[key]["race"]["skills"]:
+            skills, created = Skill.objects.get_or_create(
+                name=elem["name"],
+                defaults={"bonus": elem["bonus"], "race": race}
+            )
+        guild_data = players_data[key].get("guild")
+        if guild_data:
+            guild, created = Guild.objects.get_or_create(
+                name=guild_data["name"],
+                defaults={"description": guild_data.get("description")}
+            )
+        else:
+            guild = None
+        player, created = Player.objects.get_or_create(
+            nickname=key,
+            defaults={
+                "email": players_data[key]["email"],
+                "bio": players_data[key]["bio"],
+                "race": race,
+                "guild": guild
+
+            }
+        )
+        if player:
+            print("Player is created")
```

Comment:
The condition `if player:` is always true because `get_or_create` returns a tuple where the first element is the object and the second is a boolean indicating if it was created. Consider checking the `created` variable instead to determine if the player was newly created.